editor/src/clj/editor/model_scene.clj

```diff
@@ -355,8 +352,8 @@
      :renderable {:render-fn render-outline
                   :tags #{:model :outline}
                   :batch-key nil ; Batching is disabled in the editor for simplicity.
-                  :select-batch-key model-scene-resource-node-id
-                  :passes [pass/outline]}
+                  :select-batch-key :not-rendered ; The render-fn only does anything during the outline pass.
+                  :passes [pass/outline pass/opaque-selection]} ; Include in a selection pass to ensure it can be selected and manipulated.
```

Comment:
It seems the correct way to do this is to have the root `scene` include the `:node-id` of the `ResourceNode` (i.e. the `ModelSceneNode` in our case), and include a `:renderable` that is assigned to either `pass/selection` or `pass/opaque-selection`. It doesn't have to render anything, but this ensures it can be included in the `:selected-renderables` output of the `SceneRenderer`.

None of the `:children` scenes should specify a `:node-id`, unless they are actually from different nodes in the graph that can be transformed independently using the gizmo.